src/api/rest/server/mcis/control.go

```diff
@@ -86,8 +87,21 @@ func RestGetMcis(c echo.Context) error {
 	mcisId := c.Param("mcisId")
 
 	action := c.QueryParam("action")
+	option := c.QueryParam("option")
+
+	if option == "idOnly" || option == "idList" || option == "idListOnly" {
```

Comment:
```suggestion
	if option == "id" {
```
조금 더 간단하고 명확하게 하기 위해서,
"id" 로 처리하는 것이 어떨까요? 의미는 충분히 이해가 될 것 같습니다.